modules/platform/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -111,47 +118,47 @@ public DocumentModel createOrUpdate(FileImporterContext context) throws IOExcept
 
             ZipEntry index = zip.getEntry(MARKER);
             try (Reader reader = new InputStreamReader(zip.getInputStream(index));
-                 CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
 
                 Map<String, Integer> header = csvParser.getHeaderMap();
                 for (CSVRecord csvRecord : csvParser) {
                     String type = null;
-                    String id = null;
+                    String name = null;
                     Map<String, String> stringValues = new HashMap<>();
                     for (String headerValue : header.keySet()) {
                         String lineValue = csvRecord.get(headerValue);
                         if ("type".equalsIgnoreCase(headerValue)) {
                             type = lineValue;
-                        } else if ("id".equalsIgnoreCase(headerValue)) {
-                            id = lineValue;
+                        } else if ("name".equalsIgnoreCase(headerValue)) {
+                            name = lineValue;
                         } else {
                             stringValues.put(headerValue, lineValue);
                         }
                     }
 
+                    if (StringUtils.isBlank(name)) {
+                        log.warn("Can not create or update doc without a name, skipping line: {}",
+                                csvRecord.getRecordNumber());
```

Comment:
Not a big deal, as WARN messages will rarely be hidden, and this is not a performance hot spot, but (as some check tools warn about) the better syntax here is to use `csvRecord::getRecordNumber` to delay evaluation until the message is actually required.

Reply:
Cool, I had not thought of this. I made this update now, thanks for pointing it out.